apps/wallet-mobile/src/yoroi-wallets/cardano/shelley/ShelleyWallet.ts

```diff
@@ -602,6 +602,9 @@ export const makeShelleyWallet = (constants: typeof MAINNET | typeof TESTNET) =>
       const changeAddr = await this.getAddressedChangeAddress()
       const addressedUtxos = await this.getAddressedUtxos()
 
+      const protocolParams = await this.getProtocolParams()
+      const {coins_per_utxo_word, key_deposit, min_utxo, pool_deposit, min_fee_a, min_fee_b} = protocolParams
+
```

Comment:
not entirely sure what are all the constants that we will replace and get from the api